src/controllers/user.js

```diff
@@ -111,7 +118,67 @@ const User = {
       ));
     }
 
-    return res.status(200).send(result.toJSON());
+    const retVal = result.toJSON();
+    retVal.imgURL = imgURL;
+
+    return res.status(200).send(retVal);
+  },
+
+  update: async (req, res, next) => {
+    const { id } = req.params;
+    let result;
+
+    try {
+      result = await UserModel.findByIdAndUpdate(id, req.body, { new: true }).exec();
+    } catch (err) {
+      return next(new APIError());
```

Comment:
I think you should the error code (probably 500) here, but I am not sure

Reply:
Error code 500 is the default for `APIError`